jupyterhub_singleuser_profiles/api/api.py

```diff
@@ -16,7 +16,11 @@
 
 from jupyterhub.services.auth import HubAuth
 
-_PROFILES = SingleuserProfiles(verify_ssl=False)
+custom_notebook_namespace = None
+if 'NOTEBOOK_NAMESPACE' in os.environ:
```

Comment:
You can just do 

```custom_notebook_namespace = os.environ.get('NOTEBOOK_NAMESPACE')```

 here as `get()` returns `None` as default in case the env var does not exist